templates/addCourse.phtml

```diff
@@ -39,14 +39,20 @@ $trainersCheckbox = \Portal\ViewHelpers\TrainerCheckboxViewHelper::displayTraine
         <label for="course-notes">Notes</label>
         <textarea name="notes" class="create-courses form-control" placeholder="Notes" data-max="5000" maxlength="5000"></textarea>
         <div id="courseNotesError" class="alert hidden formItem_alert"></div>
+
         <div class="form-group">
             <label for="in_person">In Person?</label>
             <input type="checkbox" id="in_person" value="1" class="create-courses" name="in_person">
+            <input id="in_person_spaces" name="in_person_spaces" class="create-courses form-control hidden" type="text" data-required="true" />
```

Comment:
also this is missing a label? am I just supposed to psychically know what to use this input box for?